pytorch_lightning/loops/optimization/manual_loop.py

```diff
@@ -74,6 +76,10 @@ class ManualOptimization(Loop[_OUTPUTS_TYPE]):
 
     def __init__(self) -> None:
         super().__init__()
+        # since manual optimization does not track lr scheduler or optimizer frequencies, we use a simpler progress than
+        # `OptimizationProgress`
+        self.optim_step_progress = Progress.from_defaults(ReadyCompletedTracker)
```

Comment:
Personal preference but I would go for full name rather than contraction. E.g.:
```suggestion
        self.optimizer_step_progress = Progress.from_defaults(ReadyCompletedTracker)
```